graphql-ballerina/tests/utils.bal

```diff
@@ -79,12 +79,12 @@ isolated function readFileAndGetString(string filePath) returns string {
     }
 }
 
-function getJsonPayloadFromService(string url, string document) returns json|error {
+isolated function getJsonPayloadFromService(string url, string document) returns json|error {
     http:Client httpClient = check new(url);
     json payload = {
         query: document
     };
     http:Request request = new;
     request.setPayload(payload);
-    return check httpClient->post("/", request, json);
+    return check httpClient->post("/", request, targetType = json);
```

Comment:
```suggestion
    return check httpClient->post("/", request, targetType = json);
```
```suggestion
    return check httpClient->post("/", payload, targetType = json);
```

Reply:
Or else can send the inline json also rather than defining the `payload` var. From the type itself the content-type header is inferred. So in this case it will be `application/json`